docker-compose.yml

```
@@ -0,0 +1,37 @@
+version: "3.0"
+
+services:
+    front:
+        build:
+            context: .
+            dockerfile: Dockerfile.front
+        ports:
+            - 8080:8080
+        depends_on:
+            - back
+        image: gcr.io/pixie-prod/demos/px-mongo/frontend:0.1.0
+
+    back:
+        build:
+            context: .
+            dockerfile: Dockerfile.back
+        ports:
+            - 8085:8085
+        depends_on:
+            - mongodb
+        image: back:latest
+
+    load:
+        build:
+            context: .
+            dockerfile: Dockerfile.load
```

I was thinking it would be best to reuse the upstream project's Dockerfiles with minimal changes. We can get the front and back images to build unmodified if we use the `context` and `dockerfile` keys in a different way.

My interest in this is to keep our project compatible with in place change to these files. That will help us in merging in any upstream changes.

I've validated the suggested change below works for the back and frontend image building. The locust Dockerfile requires a minor change to it since it is hardcoding `./src/load-test`.

```suggestion
            context: ./src/front/
            dockerfile: ../../dockerfiles/Dockerfile.front
        ports:
            - 8080:8080
        depends_on:
            - back
        image: gcr.io/pixie-prod/demos/px-mongo/frontend:0.1.0

    back:
        build:
            context: ./src/back
            dockerfile: ../../dockerfiles/Dockerfile.back
        ports:
            - 8085:8085
        depends_on:
            - mongodb
        image: back:latest

    load:
        build:
            context: ./src/load-test
            dockerfile: ../../dockerfiles/Dockerfile.load
```